packages/models/src/eservice/eservice.ts

```diff
@@ -104,6 +107,25 @@ export const EServiceMode = z.enum([
 ]);
 export type EServiceMode = z.infer<typeof EServiceMode>;
 
+export const EServiceTemplateInterfaceInfo = z.object({
+  name: z.string(),
+  email: z.string(),
+  url: z.string(),
+  termsAndConditionsUrl: z.string(),
+  serverUrl: z.string(),
+});
+
+export type EServiceTemplateInterfaceInfo = z.infer<
+  typeof EServiceTemplateInterfaceInfo
+>;
+
+export const EServiceTemplateInfo = z.object({
```

Comment:
I'm not sure that  "Info" postfix is the best naming convention for those data definition:
```suggestion
export const EServiceTemplate = z.object({
```

Reply:
This would cause a name conflict between this and the `EServiceTemplate` model exported from `eservice-template.ts`